src/cluster/cluster.cc

```diff
@@ -214,7 +214,7 @@ Status Cluster::SetClusterNodes(const std::string &nodes_str, int64_t version, b
   if (!migrated_slots_.empty()) {
     for (const auto &[slot, _] : migrated_slots_) {
       if (slots_nodes_[slot] != myself_) {
-        auto s = srv_->slot_migrator->ClearKeysOfSlot(kDefaultNamespace, slot);
+        auto s = srv_->slot_migrator->ClearKeysOfSlotRange(kDefaultNamespace, {slot, slot});
```

Comment:
Nit: a helper function for single range is ok here

Reply:
Do you mean to keep `ClearKeysOfSlot`?

Reply:
Nope, I mean maybe two helper functions:

```
SlotRange::GetPoint()
SlotRange::GetInvalid()
```

Reply:
I will add a static function `GetPoint` in `SlotRange`. For `GetInvaild`, I feel that it may not be suitable if it is used to replace `{-1, -1}`, because sometimes `{-1, -1}` represents a wrong SlotRange, and sometimes represents an empty SlotRange, depending on the context.